Assets/Scripts/LoudObjects/LoudObject.cs

```diff
@@ -0,0 +1,90 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class LoudObject : MonoBehaviour
+{
+    public float thrust;
+    public Vector3 direction;

```

Comment:
Rename to something like, `thurstDirection` or something similar?

Reply:
I changed `public float thrust` to `public float thrustDirection`, is that what you meant?

Reply:
NEVERMIND! haha I didn't see your above comment for some reason, I understand!!